guides/common/modules/proc_synchronizing-the-system-clock-with-chronyd.adoc

```diff
@@ -13,7 +13,7 @@ For more information about the `chrony` suite, see https://access.redhat.com/doc
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install} chrony
+# {client-package-install-el8} chrony
```

Comment:
Use `project-*`?

Reply:
No change; satellite docs also show "dnf" here: https://docs.theforeman.org/nightly/Installing_Server/index-satellite.html#synchronizing-the-system-clock-with-chronyd_satellite I assume because `satellite-maintain` is not installed at this point in time; although the assembly suggests otherwise. :shrug: I'd rather not change anything here.